AK/Math.h

```diff
@@ -791,11 +791,65 @@ constexpr T atan2(T y, T x)
         : "st(1)");
     return ret;
 #else
-#    if defined(AK_OS_SERENITY)
-    // TODO: Add implementation for this function.
-    TODO();
-#    endif
-    return __builtin_atan2(y, x);
```

Comment:
I suspect that removing the __builtin_atan2 libcall for non-serenity arm64 will regress Lagom tests on macOS. But I'm happy to be proven wrong by CI :D

Reply:
It might work since `tan()` is still a libcall on non-serenity arm64.

CI won't tell us though since it doesn't run tests.

I just build Build/lagom/bin/pdf and used it to render Tests/LibPDF/paths.pdf (which exercises atan2 enough that it looks fairly broke in serenity/arm64 still) and the output looks fine. So it's probably ok :)

Reply:
(Resolved in comments below.)